src/components/NFTCover.vue

```diff
@@ -86,6 +103,8 @@ export default {
           'object-contain w-full',
           {
             'animate-pulse': !this.isLoaded,
+            'h-full min-h-[290px] w-full max-w-[200px] object-cover': this
+              .isNftBook,
```

Comment:
`w-full` is existed in line 103
seems `object-cover` and `object-contain` can be existed at the same time